server/channels/app/properties/property_field.go

```diff
@@ -19,6 +19,10 @@ func (ps *PropertyService) GetPropertyFields(ids []string) ([]*model.PropertyFie
 	return ps.fieldStore.GetMany(ids)
 }
 
+func (ps *PropertyService) CountPropertyFieldsForGroup(groupID string) (int64, error) {
+	return ps.fieldStore.CountForGroup(groupID)
+}
```

Comment:
To cover more ground and use cases, we can allow the counts to include the deleted fields:

```suggestion
func (ps *PropertyService) CountPropertyFieldsForGroup(groupID string, includeDeleted bool) (int64, error) {
	return ps.fieldStore.CountForGroup(groupID, includeDeleted)
}
```